question-2/index.html

```diff
@@ -2,21 +2,28 @@
 
 <!DOCTYPE html>
 <html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Return first and last name from a form</title>
+    <script>
+      function getFormvalue() {
+        const firstName = document.getElementsByName("fname")[0].value;
+        const lastName = document.getElementsByName("lname")[0].value;
+        const fullName = `${firstName} ${lastName}`;
+        alert(fullName);
+      }
+    </script>
+  </head>
 
-<head>
-  <meta charset=utf-8 />
-  <title>Return first and last name from a form</title>
-</head>
-
-<body>
-  <form id="form1" onsubmit="getFormvalue()">
-    First name:
-    <input type="text" name="fname" value="David">
-    <br> Last name:
-    <input type="text" name="lname" value="Beckham">
-    <br>
-    <input type="submit" value="Submit">
-  </form>
-</body>
-
-</html>
\ No newline at end of file
+  <body>
+    <form id="form1" onsubmit="getFormvalue()">
+      First name:
```

Comment:
Shouldn't `First name` be in `<label>` tag?

Reply:
I will fix this